src/Symfony/Component/Message/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message\Exception;
+
+/**
+ * Base Message component's exception.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface ExceptionInterface
```

Comment:
is there any value to add an empty interface?

Reply:
That's a good question; but it's a constant in every Symfony Component so I didn't want to start questioning this one 😉

Reply:
Using this interface, allows you to catch all the exceptions that come from this component.

Reply:
ok thanks!